packages/dashboard-hardhat-plugin/package.json

```
@@ -0,0 +1,54 @@
+{
+  "name": "@truffle/dashboard-hardhat-plugin",
+  "version": "0.1.0",
```

Oops, sorry it should be:

```suggestion
  "version": "0.1.0-0",
```

This way it gets bumped to 0.1.0 when we release